modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/Eq2K.kt

```diff
@@ -0,0 +1,53 @@
+package arrow.typeclasses
+
+import arrow.Kind2
+
+/**
+ * The `Eq2K` typeclass abstracts the ability to lift the Eq class to binary type constructors.
+ */
+interface Eq2K<F> {
```

Comment:
I think `EqK2` is a better name as I would read it as "EqualityKind2".

Reply:
yes makes sense. i have changed it